### ISSUE TITLE
Zigbee2mqtt: Add device LED1837R5

### ISSUE THREAD

**Issue by mikitouf** [GladysAssistant/Gladys#1255]
HI,
```
{
  "name": "0x804b50fffe6b749e",
  "external_id": "zigbee2mqtt:0x804b50fffe6b749e",
  "model": "LED1837R5",
  "features": [],
  "should_poll": false,
  "service_id": "9f41080f-72be-4813-b30b-0cfdd0dc71aa",
  "supported": false
}
```

**Comment by Pierre-Gilles** [comment]
Seems brightness is missing: 

https://community.gladysassistant.com/t/rajout-dune-fonctionnalite-sur-le-device-led1837r5-ikea/6548

**Comment by atrovato** [comment]
Hi @mikitouf ,
this should be fixed with Gladys 4.7.x.
Can you check:
1. Gladys has been well updated?
2. your device is now complete?

Thank you.

**Comment by mikitouf** [comment]
feature is ok
